golang.go

```diff
@@ -0,0 +1,25 @@
+// package docummentation
```

Comment:
file is not gofmted with -s (gofmt)
<!-- lookout comment separator -->
---
 file is not goimported (goimports)